src/router.js

```diff
@@ -6,20 +6,25 @@ let router = new Router({
   mode: 'history',
   routes: [{
     path: '/',
-    component: () => import('./components/homepage/home.vue')
+    component: () =>
+      import ('./components/home/page.vue')
   }, {
     path: '/user/logout',
-    component: () => import('./components/user/logout.vue')
+    component: () =>
+      import ('./components/user/logout.vue')
   }, {
     path: '/problem/list',
-    component: () => import('./components/problem/list.vue')
+    component: () =>
+      import ('./components/problem/list.vue')
   }, {
     path: '/problem/:id',
-    component: () => import('./components/problem/view.vue')
+    component: () =>
+      import ('./components/problem/content.vue')
   }, {
     path: '*',
-    component: () => import('./components/404.vue')
+    component: () =>
```

Comment:
I think origin code style in more clear.

Reply:
That's not me